docs/governance/community-guidelines/templates/features.yaml

```diff
@@ -0,0 +1,10 @@
+title: CCC <Service Category> Features
+category_id: CCC.<Service Category Abbreviation>
+type: <Parent Type of this Category>
+description: <1-3 sentence description of the service category>
+service_examples:
+  - <Nams of known cloud services that fall under this category>
```

Comment:
Having examples in feature.yaml will be confusing. For an example we are in services > databases > RDMS and if we give examples like AWS RDS, Aurora it will give a false picture to a reader that this feature set is only for AWS RDS and Aurora. Better not to have service examples in features.yaml

Reply:
applied!